eos/effects/modulebonusomnidirectionaltrackinglinkoverload.py

```diff
@@ -4,6 +4,9 @@
 # Modules from group: Drone Tracking Modules (10 of 10)
 type = "overheat"
 def handler(fit, module, context):
-    module.boostItemAttr("maxRangeBonus", module.getModifiedItemAttr("overloadTrackingModuleStrengthBonus"))
-    module.boostItemAttr("falloffBonus", module.getModifiedItemAttr("overloadTrackingModuleStrengthBonus"))
-    module.boostItemAttr("trackingSpeedBonus", module.getModifiedItemAttr("overloadTrackingModuleStrengthBonus"))
+    overloadBonus = module.getModifiedItemAttr("overloadTrackingModuleStrengthBonus")
+    module.boostItemAttr("maxRangeBonus", overloadBonus)
+    module.boostItemAttr("falloffBonus", overloadBonus)
+    module.boostItemAttr("trackingSpeedBonus", overloadBonus)
+    module.boostItemAttr("aoeCloudSizeBonus", overloadBonus)
+    module.boostItemAttr("aoeVelocityBonus", overloadBonus)
```

Comment:
Where are you getting
```
aoeCloudSizeBonus
aoeVelocityBonus
```
from?

The info says:
```
Improves the optimal range, falloff range, explosion radius and explosion velocity of all fighter weapons.
```

Looking at a Templar, I would think that the stats needed to be modified would be:

```
fighterAbilityAttackMissileExplosionRadius
fighterAbilityAttackMissileExplosionVelocity
fighterAbilityMissilesExplosionRadius
fighterAbilityMissilesExplosionVelocity
```
The other fighters might have different stats to modify.